.fernignore

```diff
@@ -4,3 +4,7 @@ src/humanloop/eval_utils.py
 src/humanloop/prompt_utils.py
 src/humanloop/client.py
 mypy.ini
+
+# Directories used by SDK decorators
```

Comment:
Do I also have to add the test directories for my changes?

Reply:
Please ask fern how testing should be included. Tests should run as part of the CI on SDK publication

Reply:
https://humanloop.slack.com/archives/C06SF3Q6F7H/p1730986945280089

Asked Fern, will monitor the discussion